package.json

```diff
@@ -65,13 +65,13 @@
     "autoprefixer": "^9.4.6",
     "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^10.0.1",
-    "babel-jest": "^23.6.0",
+    "babel-jest": "^24.1.0",
     "eslint": "5.12.0",
     "eslint-config-prettier": "^3.6.0",
     "eslint-plugin-prettier": "^3.0.1",
     "eslint-plugin-standard": "^4.0.0",
     "eslint-plugin-vue": "^5.1.0",
-    "jest": "^23.6.0",
+    "jest": "^24.1.0",
```

Comment:
After we merge in this PR, #336 will give a conflict. I'll fix that one after this one is merged.